bin/run-tests-in-docker.sh

```diff
@@ -28,4 +28,4 @@ docker run \
     --volume "${PWD}/bin/run-tests.sh:/opt/test-runner/bin/run-tests.sh" \
     --workdir /opt/test-runner \
     --entrypoint /opt/test-runner/bin/run-tests.sh \
-    exercism/test-runner
+    exercism/idris-test-runner
```

Comment:
Same issue as my other comment looking at how https://github.com/exercism/lfe-test-runner/blob/017f86a6c1d8c844a3531e8bd0613c10b537d8f5/bin/run-in-docker.sh#L44 is written